css/wishlist.css

```diff
@@ -0,0 +1,72 @@
+.wishlist-container {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(280px, 1fr));
+    padding: var(--size-3);
+    grid-column-gap: var(--size-0-5);
+    grid-row-gap: var(--size-1);
+    margin: auto;
+}
+
+.wishlist-container .card.card-vertical {
+    background-color: var(--white-bg);
+    max-width: 30rem;
+    display: flex;
+    width: 21rem;
+    box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.2), 15px 17px 60px var(--grey-bg);
+}
+
+.wishlist-container .card.card-vertical .vt-card-img {
+    width: 100%;
+    height: 21rem;
+    object-fit: cover;
+}
+
+.wishlist-container .card.card-vertical h2 {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 500;
+}
+
+.card-brand-name {
+    font-weight: 300;
+}
+
+.wishlist-container .card.card-vertical .card-vt-btn {
+    display: flex;
+    justify-content: space-around;
+}
+
+.wishlist-container .card.card-vertical .card-vt-btn .button.button-primary {
+    min-width: 30%;
+    width: 60%;
+    font-size: 1.2rem;
+    display: flex;
+    justify-content: center;
+}
+
+.wishlist-container .card.card-vertical .card-vt-btn .button.button-secondary {
+    min-width: 10%;
+    width: 20%;
+    font-size: 1.2rem;
+    height: 100%;
+}
+
+.wishlist-container .card.card-vertical .card-vt-btn .button .fas,
+.wishlist-container .card.card-vertical .card-vt-btn .button .fab,
+.wishlist-container .card.card-vertical .card-vt-btn .button .far {
+    margin: 0;
```

Comment:
suggestion- you can make utility classes for this in your component lib for ease

Reply:
Will do this at the end thanks!